### PR TITLE
NorKyst800 experiments

### PR DESCRIPTION
**Pull request by havahol**
Contains notebook with all relevant cases and plotting for real-world simulations.

@babrodtk: You should be able to run the NorKyst800_Experiments.ipynb notebook as is in order to get timing results.

@setmar: Let me know if this PR comes in conflict with your work (potential conflicts on RealisticSimulation.ipynb and BC_tests.ipynb)

### CODE REVIEW

File: gpu_ocean/SWESimulators/Common.py [hunk 1]
```diff
@@ -212,6 +212,10 @@ def __exit__(self, *args):
     def getPrintString(self, x):
         elapsed =  time.time() - self.start
         
+        if (x == 0):
+            self.print_string = ProgressPrinter.formatString(0, 0, np.nan)
```
Comment (babrodtk) [inline gpu_ocean/SWESimulators/Common.py]: Bedre å bruke None her

File: gpu_ocean/SWESimulators/Common.py [hunk 2]
```diff
@@ -243,6 +247,9 @@ def formatString(t, elapsed, remaining_time):
                 
 
     def timeString(seconds):
+        if np.isnan(seconds):
```
Comment (babrodtk) [inline gpu_ocean/SWESimulators/Common.py]: if seconds is None

File: gpu_ocean/demos/realisticSimulations/BasicSim.ipynb [hunk 3]
```diff
@@ -8,7 +8,7 @@
     "This notebook sets up and runs a set of benchmarks to compare\n",
```
Comment (babrodtk) [inline gpu_ocean/demos/realisticSimulations/BasicSim.ipynb]: Suggest deleting this file - it has no value any more

Comment (havahol) [reply gpu_ocean/demos/realisticSimulations/BasicSim.ipynb]: I wish to keep it, as it is a visual reminder of the availability of the Norden domain :) I renamed it accordingly.

File: gpu_ocean/SWESimulators/Common.py (hunk 1 above)
Comment (havahol) [reply gpu_ocean/SWESimulators/Common.py]: Det ble verre å bruke None. Mange operasjonene som brukes er veldefinerte for NaN, men ikke for None (+, max etc). Fortsetter derfor med np.nan.

**Pull request merged**